packages/core/admin/admin/src/components/UnstableGuidedTour/Step.tsx

```diff
@@ -49,14 +49,23 @@ const ActionsContainer = styled(Flex)`
   border-top: ${({ theme }) => `1px solid ${theme.colors.neutral150}`};
 `;
 
+const PopoverArrowStandard = styled(Popover.Arrow)`
+  fill: ${({ theme }) => `${theme.colors.neutral0}`};
+`;
+
 const createStepComponents = (tourName: ValidTourName): Step => ({
-  Root: React.forwardRef((props, ref) => (
-    <Popover.Content ref={ref} side="top" align="center" style={{ border: 'none' }} {...props}>
-      <Flex width="360px" direction="column" alignItems="start">
-        {props.children}
-      </Flex>
-    </Popover.Content>
-  )),
+  Root: React.forwardRef((props, ref) => {
+    const state = unstableUseGuidedTour('GuidedTourPopover', (s) => s.state);
+    const currentStep = state.tours[tourName].currentStep + 1;
+    return (
+      <Popover.Content ref={ref} side="top" align="center" {...props}>
+        <Flex width="360px" direction="column" alignItems="start">
+          {props.children}
+        </Flex>
+        {currentStep > 1 && <PopoverArrowStandard width="20px" height="12px" />}
```

Comment:
I see what you did there, makes sense for our current setup but I think still would be better to either
- add a prop like `withArrow` that defaults to true since most cases have the arrow, and then set it to false on the ones that don't have it
- rely on composition and add the PopoverArrow directly in the steps that need it